composer.json

```diff
@@ -51,7 +51,7 @@
         "psr/http-message": "^1.0",
         "ramsey/uuid": "^4.0",
         "google/gax": "^1.12",
-        "google/common-protos": "^3.0",
+        "google/common-protos": "^3.0||^4.0",
```

Comment:
Would it make sense to just have `^4.0`?

Reply:
Yes, I think just `4.0` would be better. I'll make the change!

Reply:
```suggestion
        "google/common-protos": "^4.0",
```